### DISCUSSION TITLE
Scan interval stability

### DISCUSSION THREAD

**Discussion by TobiasLaross** [TobiasLaross/Hass-Lynk-Co#31] [General]
As discussed in #23 - What have you set the scan interval to and how stable has it been for you? I have used 240min without issues and refreshed manually when I needed data. Still, I lowered it to 120 minutes yesterday because I added automation to send push notifications for time until fully charged.

**Comment by pjgrun** [comment]
I am using 60 minutes and is working fine for now.
I just installed new version 1.4.0.

**Comment by pjgrun** [comment]
BTW, I tested to login into Lynk&Co official app while keeping HA Integration operational. It completed log in and during aprox 15 minutes I was able to access in parallel from both, HA Integration and official App. Then I was kicked out from official app.

**Comment by TobiasLaross** [comment]
I have tested using the official app a couple of times and it seems to be maximum of 24h (life time of cccToken) but mostly shorter. I don't know why they enforce it but it is the same (or similar) as sharing your account with your partner which was the biggest reason for me to create this in the first place.
Is there some functionality from the official app you are missing?

**Reply by pjgrun** [reply]
Is there some functionality from the official app you are missing? -> Digital key

**Reply by TobiasLaross** [reply]
Does that let you drive the car without physical key? I would assume that it would require a ble connection between a native app and car unfortunately

**Reply by pjgrun** [reply]
Right, digital key functionality supported by official app lets you open and drive car without a physical key present. And as you anticipate, it uses Bluetooth connection between app and car.

**Comment by SYSKOMM** [comment]
Hello,
manual updating does not work.
the 60 minute interval doesn't work either.

**Reply by TobiasLaross** [reply]
> Hello, manual updating does not work. the 60 minute interval doesn't work either.

Do you have any errors in the logs?

**Reply by SYSKOMM** [reply]
2024-05-27 08:53:08.529 WARNING (MainThread) [homeassistant.components.automation.force_update_lynk] Force Update Lynk: Already running

**Reply by SYSKOMM** [reply]
![Update](https://github.com/TobiasLaross/Hass-Lynk-Co/assets/76566876/d794e00c-d10b-4521-b241-ccd325217897)

**Reply by TobiasLaross** [reply]
That is your automation named force_update_lynk that isn't working, look for custom_components.lynkco in the logs.
Force update will only update the data available from lynk backend, it will not force the car to push data to the backend

**Comment by pjgrun** [comment]
Hi, since a few days I am testing forcing updates every 15 minutes. It is working fine so far.

**Reply by TobiasLaross** [reply]
Great, it sounds like the servers are unstable sometimes and not directly tied to refresh rate

**Reply by SYSKOMM** [reply]
still doesn't work for me

**Comment by poliant** [comment]
I installed the custom components less that 24h ago with refresh rate at standard 120mins and tried many times the "force_update" and no issue till now. I would monitor in next days and provide feedback

question: is it possible to disable the "dark hours"? I set start and end at same hour (e.g. 1) and will test in next days to chec the behaviour

**Reply by TobiasLaross** [reply]
There is actually a ticket raised that dark hours doesn’t work #38 , but I can add an option to disable dark hours when ever I get around to fix it
